### ISSUE TITLE
(question) How to configure electrode locations/coordinate frame

### ISSUE THREAD

**Issue by droyston** [beauchamplab/rave#47]
I am trying to add electrode coordinates in the electrodes.csv meta file, using the individual-space coordinates compatible with the Freesurfer reconstruction, but the RAVE render shows all electrodes at a single point near the origin of the render space. Which coordinate frame/range of locations does RAVE expect?

**Comment by jmagnotti** [comment]
Can you post the electrodes.csv file that you are using? If you're using freesurfer locations, they should work

**Comment by dipterix** [comment]
Thanks @droyston , 

This is a good question.  We are using `tkrRAS` coordinate.

Please check [freesurfer document](https://surfer.nmr.mgh.harvard.edu/fswiki/CoordinateSystems) on how to transform from scanner or MNI305 to tkrRAS.

`TalXFM`, `Torig` and `Norig` can be obtained via `rave` code in R:

``` r
# Change project/subject to project and subject
brain = rave::rave_brain2("project/subject")

# Talairach
brain$xfm

# Norig
brain$Norig

# Torig
brain$Torig
```

**Comment by droyston** [comment]
> 
> 
> Can you post the electrodes.csv file that you are using? If you're using freesurfer locations, they should work

Yes I can, the file is attached (converted to txt because the comment doesn't appear to support csv)

[electrodes.txt](https://github.com/beauchamplab/rave/files/5638554/electrodes.txt)

**Comment by jmagnotti** [comment]
These values are really tiny... how did you extract these? Perhaps they are fractional values of the total space? You could try multiplying them by 256 and see if that fixes the problem

**Comment by droyston** [comment]
> 
> 
> These values are really tiny... how did you extract these? Perhaps they are fractional values of the total space? You could try multiplying them by 256 and see if that fixes the problem

These coordinates are extracted from the coregistered channel objects in Brainstorm, and appear to align correctly in Brainstorm's surface-viewer

**Comment by dipterix** [comment]
Hmm, not sure how brainstorm handles coordinate. They have their own coordinate system in the Brainstorm MRI structure files. However, they can be transformed to CRS, MRI, MNI via its [toolbox](https://github.com/brainstorm-tools/brainstorm3/blob/master/toolbox/anatomy/cs_convert.m).
